src/openrct2/ride/ride.c

```diff
@@ -5689,6 +5689,16 @@ void game_command_set_ride_name(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *e
     }
 
     *ebx = 0;
+
+	//After renaming a ride, refresh Guest list if open
+	rct_window* window;
+
+	window = window_find_by_class(WC_GUEST_LIST);
+	if (window == NULL) {
+		return;
```

Comment:
This would be much clearer if the sense of the condition was inverted:
```
if (window != NULL)
{
window_guest_list_find_groups_force_refresh();
}
```